tests/pipelines/kandinsky2_2/test_kandinsky_prior.py

```diff
@@ -235,3 +237,41 @@ def test_attention_slicing_forward_pass(self):
             test_max_difference=test_max_difference,
             test_mean_pixel_difference=test_mean_pixel_difference,
         )
+
+    def test_callback_inputs(self):
```

Comment:
I think we would need one test to for tensor inputs and one for other properties (cfg, cross attention kwargs, etc)